src/woo_publications/publications/models.py

```diff
@@ -76,6 +78,12 @@ class Publication(models.Model):
         blank=True,
     )
     omschrijving = models.TextField(_("description"), blank=True)
+    publicatiestatus = models.CharField(
+        _("status"),
+        max_length=12,
+        choices=PublicationStatusOptions.choices,
+        default=PublicationStatusOptions.published,
```

Comment:
is that a sane default? :thinking: let's discuss with Marco

Reply:
It definitely is not but it was the described behavior. I think it would indeed be good to discuss this with Marco.

Reply:
discussed and decided to keep this default - changing it later is a minor task so no worries here